src/sparseml/transformers/compression/compressors/sparse_bitmask.py

```diff
@@ -54,46 +61,77 @@ def compress(self, model_state: Dict[str, Tensor]) -> Dict[str, Tensor]:
             f"Compressing model with {len(model_state)} parameterized layers..."
         )
         for name, value in tqdm(model_state.items()):
-            bitmask_tensor = BitmaskTensor(value)
+            bitmask_tensor = BitmaskTensor.from_dense(value)
             compressed_dict |= bitmask_tensor.dict(name_prefix=name)
 
         return compressed_dict
 
-    def decompress(self, model_state: Dict[str, Tensor]) -> Dict[str, Tensor]:
+    def decompress(self, model_path: str) -> Dict[str, Tensor]:
         """
-        Uncompresses a bitmask compressed state dict back to dense
+        Reads a bitmask compressed state dict located at model_path and decompresses it
+        back to a dense state dict. Weights are decompressed sequentially.
 
-        :param model_state: state dict of uncompressed model
+        :param model_path: path to compressed safetensors model
         :return: compressed state dict
         """
-        raise NotImplementedError()
+        weight_mappings = get_nested_weight_mappings(
```

Comment:
The decompression will fail if the expected keys aren't provided, we're assuming if decompress is called its because we've inferred this bitmask compression format from the config file (loading from the config file will be addressed in a follow up PR)